features/mbedtls/targets/TARGET_CRYPTOCELL310/sha256_alt.c

```diff
@@ -0,0 +1,86 @@
+/*
+ *  sha256_alt.c
```

Comment:
Same questions/comments in this module as with SHA1

Reply:
same answers